code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -411,12 +411,25 @@
 	// They stay locked down if their wire is cut.
 	if(wires?.is_cut(WIRE_LOCKDOWN))
 		state = TRUE
+	else
+		if(!ai_lockdown)
+			addtimer(CALLBACK(src,PROC_REF(lockdown_override), FALSE), 10 MINUTES)
```

Comment:
We're scheduling an event 10 minutes in the future and then just letting it run?
This timer should probably be stored in a var and clear itself if the cyborg is released (or dies?) during that time period instead of waiting the full 10 minutes before checking and then giving a weird error message which will no longer have any context to the player.
Also it should definitely kill off the timer when this mob is deleted, which I don't think it currently does 😅

Reply:
I changed it, can you look at it now?